Mathlib/Topology/Algebra/Module/Basic.lean

```diff
@@ -142,12 +142,10 @@ theorem Submodule.mapsTo_smul_closure (s : Submodule R M) (c : R) :
     Set.MapsTo (c • ·) (closure s : Set M) (closure s) :=
   have : Set.MapsTo (c • ·) (s : Set M) s := fun _ h ↦ s.smul_mem c h
   this.closure (continuous_const_smul c)
-#align submodule.maps_to_smul_closure Submodule.mapsTo_smul_closure
```

Comment:
Please don't remove the `#align` statements.

Reply:
Oh, my bad.  I didn't realize they were removed from mathlib3 as well: https://github.com/leanprover-community/mathlib4/pull/2983#discussion_r1148313871